BookReader/BookReader.js

```diff
@@ -2798,27 +2798,31 @@ BookReader.prototype.flipRightToLeft = function(newIndexL, newIndexR) {
 BookReader.prototype.setMouseHandlers2UP = function() {
     var self = this;
     ['L', 'R'].forEach(function(dir) {
-      var currentIndexEl = self.prefetchedImgs[self.twoPage['currentIndex' + dir]];
-      self.setClickHandler2UP(currentIndexEl,
-          { self: self },
-          function(e) {
-              if (e.which == 3) {
-                  // right click
-                  return !e.data.self.protected;
-              }
+        var currentIndexEl = self.prefetchedImgs[self.twoPage['currentIndex' + dir]];
+        self.setClickHandler2UP(currentIndexEl,
+            { self: self },
+            function(e) {
+                if (e.which == 3) {
+                    // right click
+                    return !e.data.self.protected;
+                }
 
-              // Changes per WEBDEV-2737
-              // BookReader: zoomed-in 2 page view, clicking page should change the page
-              $(currentIndexEl)
-              .mousemove(function() {
-                  e.preventDefault();
-              })
-              .mouseup(function() {
-                  e.data.self.trigger(BookReader.eventNames.stop);
-                  e.data.self.left();
-              });
-          }
-      );
+                // Changes per WEBDEV-2737
+                // BookReader: zoomed-in 2 page view, clicking page should change the page
+                $(currentIndexEl)
+                .mousemove(function() {
+                    e.preventDefault();
+                })
+                .mouseup(function() {
+                    e.data.self.trigger(BookReader.eventNames.stop);
+                    if (dir === 'L') {
```

Comment:
That's odd. I must not have copied that change over to the committed code. It's in the file on the testing vm-home instance. What I did was a one-liner like
`e.data.self[dir === 'L' ? 'left' : 'right']()`

Reply:
Changed per one-liner suggestion. Thanks @shaneriley!